petstore/src/test/scala/io/finch/petstore/package.scala

```diff
@@ -0,0 +1,69 @@
+package io.finch
+
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.{Arbitrary, Gen}
+
+package object petstore {
+  implicit val statusArbitrary: Arbitrary[Status] =
+    Arbitrary(Gen.oneOf(Available, Pending, Adopted))
+
+  implicit val orderStatArbitrary: Arbitrary[OrderStatus] =
+    Arbitrary(Gen.oneOf(Placed, Approved, Delivered))
+
+  implicit val categoryArbitrary: Arbitrary[Category] = Arbitrary(
+    for {
+      id <- arbitrary[Long]
+      name <- Gen.alphaStr
+    } yield Category(Option(id), name)
```

Comment:
Maybe do a search for all instances of `Option(...)` and change to `Some(...)` if the value can never be `null` (or unless necessary for type inference, which shouldn't be the case in most places here).